input/pagecontent/general-requirements.md

```diff
@@ -315,28 +143,40 @@ The full set of sub-elements is visible in the "Key Elements Table" or "Snapshot
 
-- AU Core Responders **SHALL** be capable of providing a value in `Organization.name`.
-- AU Core Requestors **SHALL** be capable of processing an Organization resource with a value in `Organization.name`.
+- AU Core Responder **SHALL** correctly populate a value in `Organization.name` if a value is know.
+- AU Core Requester **SHALL** accept an Organization resource without error if `Organization.name` is present and containing any valid value.
 
 ##### Must Support - Complex Elements
 Complex elements are composed of primitive and/or other complex elements. Elements may have additional rules defined in the profile that also apply, e.g. terminology binding, or invariants. 
 
-For any complex element labelled as *Must Support*, an AU Core Responder **SHALL** be capable of providing at least one of the sub-element values. For some complex types a valid value can be constructed by populating only one sub-element, but usually more than one sub-element is needed. An AU Core Requestor **SHALL** be capable of processing the resource with all sub-elements.
+For any complex element labelled as *Must Support*, an:
+- AU Core Responder **SHALL** correctly populate the element with at least one of the sub-element values.
+- AU Core Requester **SHALL** accept resource without error if the element is present and containing any sub-elements.
 
 For example, the AU Core MedicationRequest Profile `note` element is labelled *Must Support* and has no *Must Support* sub-elements. When claiming conformance to this profile:
-- AU Core Responders **SHALL** be capable of providing a value in a `MedicationRequest.note` sub-element e.g. `MedicationRequest.note.text`.
-- AU Core Requestors **SHALL** be capable of processing the MedicationRequest resource with a value in `MedicationRequest.note`.
+- AU Core Responder **SHALL** correctly populate a value in a `MedicationRequest.note` sub-element if a value is known e.g. `MedicationRequest.note.text`.
+- AU Core Requester **SHALL** accept the MedicationRequest resource without error if `MedicationRequest.note` is present and containing any sub-elements.
 
-For example, if AU Core Patient Profile name element is labelled *Must Support* and has *Must Support* sub-elements "family” and “given”. When claiming conformance to this profile:
-- AU Core Responders **SHALL** be capable of providing a value in `Patient.name.family` and `Patient.name.given`.
-- AU Core Requestors **SHALL** be capable of processing a Patient resource with a value in `Patient.name.family` and `Patient.name.given`.
+If any sub-element is labelled as *Must Support*, an: 
+- AU Core Responder **SHALL** correctly populate the element with all _Must Support_ sub-elements if a value is known. 
+- AU Core Requester **SHALL** accept resource without error if _Must Support_ sub_elements are present containing any valid value.
```

Comment:
typo "sub_elements", change to "sub-elements"

Reply:
fixed.